src/wp-admin/includes/ajax-actions.php

```diff
@@ -3568,6 +3568,18 @@ function wp_ajax_query_themes() {
 
 	$update_php = network_admin_url( 'update.php?action=install-theme' );
 
+	$installed_themes = search_theme_directories();
+
+	if ( false === $installed_themes ) {
+		$installed_themes = array();
+	}
+
+	foreach ( $installed_themes as $theme_slug => $theme_data ) {
+		if ( false !== strpos( $theme_slug, '/' ) ) {
```

Comment:
Should we use `str_contains()` here for better readability?

I think this fragment could also use a comment to clarify its purpose and explain why we're skipping theme slugs with a slash here.

Reply:
Thank you for the review.
Fixed in ab1a082cbcd51453ec00ac705e857da156a56f66